pkg/core/gitea/client.go

```diff
@@ -12,31 +12,54 @@ import (
 )
 
 type GiteaAdminClient interface {
-	PrepareRepo(code *codeset.Codeset) error
-	GetRepos(org, label *string) ([]*codeset.Codeset, error)
-	GetRepo(org, name string) (*codeset.Codeset, error)
+	PrepareRepository(code *codeset.Codeset) error
+	GetRepositories(org, label *string) ([]*codeset.Codeset, error)
+	GetRepository(org, name string) (*codeset.Codeset, error)
+}
+
+type someGiteaClient interface {
```

Comment:
Could you name this differently please ? as far as this go package is concerned, this _is_ a giteaClient, so why not name it so ?

Reply:
OK